### PR TITLE
Add stage-release workflow

### PR DESCRIPTION
**Pull request by aiyengar2**
Adds a step to call stage-release after https://github.com/rancher/charts/pull/1126 is merged

### CODE REVIEW

**Comment by aiyengar2** [comment]
Needs to be forward-ported to dev-v2.6-source as well, but will do once this has been approved.

When the original PR in https://github.com/rancher/charts/pull/1125 is forward ported to dev-v2.6, I'll open a separate PR to update dev-v2.5-source's push-2.6 workflow to also regenerate the released/ directory

File: .github/workflows/push.yaml [hunk 1]
```diff
@@ -28,3 +28,12 @@ jobs:
 
     - name: Push assets
       run: git push origin dev-v2.5
```
Comment (thedadams) [inline .github/workflows/push.yaml:30]: Should we only push once?

**Comment by thedadams** [comment]
Needs rebase, but looks good to me.

**Review by brendarearden — APPROVED** [review]
lgtm!

**Pull request merged**